pwa-devdocs/src/pwa-buildpack/reference/targetables/TargetableSet/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: TargetableSet
+---
+
+<!--
+The reference doc content is generated automatically from the source code.
+To update this section, update the doc blocks in the source code
+-->
+
+{% include auto-generated/pwa-buildpack/lib/WebpackTools/targetables/TargetableSet.md %}
+
+## Examples
+
+Code examples for using the `TargetableSet` class.
+
+### Import the class
+
+This class is available as a named import from `@magento/pwa-buildpack`.
+
+```js
+const { Targetables } = require('@magento/pwa-buildpack')
```

Comment:
Can you mention that `Targetables` and `TargetableSet` are the same because nowhere in the following examples we are using `TargetableSet`?